env/mistral.dkrz.de/shell-intel+openmpi

```diff
@@ -0,0 +1,50 @@
+# make the contents as shell agnostic as possible so we can include them with bash, zsh and others
+
+module load gcc/4.8.2
+export LD_LIBRARY_PATH=/sw/rhel6-x64/gcc/gcc-4.8.2/lib64:$LD_LIBRARY_PATH # avoid GLIBCXX_3.4.15 not found error
+module unload intel && module load intel/18.0.4
+
+export FC=mpif90 CC=mpicc CXX=mpicxx; module unload intelmpi; module load openmpi/2.0.2p2_hpcx-intel14
+
+# from https://www.dkrz.de/up/systems/mistral/running-jobs/mpi-runtime-settings
+export I_MPI_FABRICS=shm:dapl
+export I_MPI_FALLBACK=disable
+export I_MPI_SLURM_EXT=0 # disable optimized startup algorithm for intel MPI
+export I_MPI_LARGE_SCALE_THRESHOLD=8192 # set to a value larger than the number of MPI-tasks used !!!
+export I_MPI_DYNAMIC_CONNECTION=1
+export I_MPI_CHECK_DAPL_PROVIDER_COMPATIBILITY=0
+export I_MPI_HARD_FINALIZE=1
+export I_MPI_ADJUST_GATHER=1 # do not use =3 (Shumilin's algorithm)
+
+export MXM_LOG_LEVEL=ERROR # try to disable "Conflicting CPU frequencies detected" messages from OpenMPI
+
+module unload netcdf && module load netcdf_c/4.3.2-gcc48
+module unload cmake && module load cmake
+# we will get a segfault at runtime if we use a gcc from any of the provided gcc modules
+export PATH=/sw/rhel6-x64/gcc/binutils-2.24-gccsys/bin:${PATH}
+
+export NETCDF_Fortran_INCLUDE_DIRECTORIES=/sw/rhel6-x64/netcdf/netcdf_fortran-4.4.2-intel14/include
+export NETCDF_C_INCLUDE_DIRECTORIES=/sw/rhel6-x64/netcdf/netcdf_c-4.3.2-intel14/include
+export NETCDF_CXX_INCLUDE_DIRECTORIES=/sw/rhel6-x64/netcdf/netcdf_cxx-4.2.1-gcc48/include
+
+export HDF5_C_INCLUDE_DIRECTORIES=/sw/rhel6-x64/hdf5/hdf5-1.8.14-threadsafe-intel14/include
+
+
+
+### not sure if the following settings are really necessary to run the coupled awicm3
+
+export GRIBROOT=/pf/a/a270092/ecmwf/grib_api_intel_modulegcc
+export NETCDFFROOT=/sw/rhel6-x64/netcdf/netcdf_fortran-4.4.3-intel14
+export NETCDFROOT=/sw/rhel6-x64/netcdf/netcdf_c-4.4.0-gcc48
+export HDF5ROOT=/sw/rhel6-x64/hdf5/hdf5-1.8.14-threadsafe-gcc48
+export SZIPROOT=/sw/rhel6-x64/sys/libaec-0.3.2-gcc48
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${NETCDFFROOT}/lib:${HDF5ROOT}/lib:${NETCDFROOT}/lib:${SZIPROOT}/lib:${GRIBROOT}/lib
+
+export FESOM_USE_CPLNG='active'
+
+export DR_HOOK=1
```

Comment:
Are we using DR_HOOK in FESOM2 now? DR_HOOK is the IFS internal debug program. See: https://confluence.ecmwf.int/download/attachments/19661682/drhook.pdf?version=1&modificationDate=1443002579511&api=v2

If we don't use it, we should not set any flag for it here.